elasticsearch/_async/transport.py

```diff
@@ -44,7 +46,25 @@ class AsyncTransport(Transport):
 
     DEFAULT_CONNECTION_CLASS = AIOHttpConnection
 
-    def __init__(self, hosts, *args, sniff_on_start=False, **kwargs):
+    def __init__(
+        self,
+        hosts,
+        connection_class=None,
+        connection_pool_class=ConnectionPool,
+        host_info_callback=get_host_info,
+        sniff_on_start=False,
+        sniffer_timeout=None,
+        sniff_timeout=0.1,
+        sniff_on_connection_fail=False,
+        serializer=JSONSerializer(),
+        serializers=None,
+        default_mimetype="application/json",
+        max_retries=3,
+        retry_on_status=(502, 503, 504),
+        retry_on_timeout=False,
+        send_get_body_as="GET",
```

Comment:
This is missing a single parameter `meta_header`, can you also add this parameter in `elasticsearch/_async/transport.pyi`?